homeassistant/components/bluetooth_le_tracker/device_tracker.py

```diff
@@ -178,6 +179,9 @@ def _async_update_ble(
         """Update from a ble callback."""
         mac = service_info.address
         if mac in devs_to_track:
+            if mac in devs_times and service_info.time == devs_times[mac]:
+                return
+            devs_times[mac] = service_info.time
```

Comment:
Move this into `_async_refresh_ble` so we don't have to check for ones we already know are fresh

Reply:
@ProtoxiDe22 If you cherry-pick this commit https://github.com/bdraco/home-assistant/commit/71eeb229daceace84907a1235c9c0048c526fa6f and everything still works as you expect.  We would just need a test and this should be good to merge.

Reply:
Thank you, i'm testing the commit right now, it seems fine so far, i'm looking into writing a test but as I said i have never done that and i'm not sure i'll be able to, i'll report back in a few hours i think